mne/surface.py

```diff
@@ -1906,24 +1885,101 @@ def warp_montage_volume(montage, base_image, reg_affine, sdr_morph,
     return montage_warped, image_from, image_to
 
 
-def _get_neighbors(loc, image, thresh, voxels):
+_VOXELS_MAX = 100  # define constant to avoid runtime issues
+
+
+@fill_doc
+def get_montage_rois(montage, subject, subjects_dir=None,
+                     aseg='aparc+aseg', dist=5):
+    """Get the regions of interest near the channels in a montage.
```

Comment:
Say where the ROIs / volume labels come from. Say it’s coming from a freesurfer volume parcellation